### PR TITLE
feat: update Danish translation for ace

### PR DESCRIPTION
**Pull request by ddfreiling**
- Added missing Danish translations for ace-core

### CODE REVIEW

**Comment by ddfreiling** [comment]
@MyDK any interest in continuing your excellent proofreading? :)

**Comment by ddfreiling** [comment]
> > @MyDK any interest in continuing your excellent proofreading? :)
> 
> Hi I have lost access to my original github account @MyDK, so I was forced to create a new one :/ - Eg. in "testdesc": "Sikrer at xxx and ""testdesc": "Tjek at xx" I think I would write "Sørg for xxx" if the intention is call to action, like you should -do this- to fix the issue. Then you can use that phrase to all variable.

No worries about the accounts.
Omkring `Sikrer` vs `Tjek`, så har det været et forsøg på at oversætte brugen af `Ensures`, som bruges i den engelske version, se https://github.com/daisy/ace/blob/master/packages/ace-core/src/l10n/locales/en.json
Selve `testdesc` er en beskrivelse af hvad testens formål er, så vidt jeg har forstået.

File: packages/ace-core/src/l10n/locales/da.json [hunk 1]
```diff
@@ -37,22 +37,22 @@
             "testdesc": "Sikrer at '{{name}}' metadata-elementet er til stede"
         },
         "missingpagebreakviolation": {
-            "kbtitle": "",
-            "resdesc": "",
-            "ruledesc": "",
-            "testdesc": ""
+            "kbtitle": "Sidenavigation",
+            "resdesc": "Sikrer at sidelisten indeholder alle sideskift ({{ref}})",
```
Comment (MyGulstad) [inline packages/ace-core/src/l10n/locales/da.json:41]: "Sørg for, at sidelisten indeholder alle sideskift ({{ref}})",

File: packages/ace-core/src/l10n/locales/da.json [hunk 2]
```diff
@@ -37,22 +37,22 @@
...
+            "ruledesc": "Indholdsfortegnelse skal være magen til læserækkefølgen ({{ref}})",
+            "testdesc": "Sikrer at publikationens indholdsfortegnelse er i korrekt rækkefølge ({{ref}})"
         },
         "pagelistbrokenviolation": {
-            "kbtitle": "",
-            "resdesc": "",
-            "ruledesc": "",
-            "testdesc": ""
+            "kbtitle": "Sidenavigation",
+            "resdesc": "Sørg for at alle elementer på sidelisten peger på epub:type sideskift ({{ref}})",
```
Comment (MyGulstad) [inline packages/ace-core/src/l10n/locales/da.json:53]: "Sørg for, at alle elementer i sidelisten peger på epub:type sideskift ({{ref}})",

File: packages/ace-core/src/l10n/locales/da.json [hunk 3]
```diff
@@ -37,22 +37,22 @@
...
+            "testdesc": "Tjek at alle sideskift er med i sidelisten ({{ref}})"
         },
         "ordertocviolation": {
-            "kbtitle": "",
-            "resdesc": "",
-            "ruledesc": "",
-            "testdesc": ""
+            "kbtitle": "Indholdsfortegnelse",
+            "resdesc": "Rækkefølge for elementer i indholdsfortegnelsen ({{ref}})",
+            "ruledesc": "Indholdsfortegnelse skal være magen til læserækkefølgen ({{ref}})",
```
Comment (MyGulstad) [inline packages/ace-core/src/l10n/locales/da.json:48]: "Indholdsfortegnelsen skal være magen til læserækkefølgen ({{ref}})",

**Comment by ddfreiling** [comment]
Thanks @MyGulstad , PR is now ready for review/merge.

**Pull request merged**